### PR TITLE
chore: update images and instructions to 17.0

### PR DESCRIPTION
**Pull request by Meng-Heng**
Please approve this PR #1147 before reviewing this.

Sections that need experts to add information or give me context:
1. context/model-editor ✅ 
    - #details-tab is missing some instruction
2. context/new-project ✅ 
    - LDML keyboard needs an instruction
3. context/options (another PR)
    - Server tab needs an update
4. context/startup (deleted)

### CODE REVIEW

File: developer/17.0/context/debug.md [hunk 1]
```diff
@@ -62,15 +62,13 @@ Run
...
-    Press Pause again to resume debugging.
+:   Use the Pause button or press <kbd>Shift</kbd> + <kbd>Esc</kbd> to pause the debugger. When
+    the debugger is paused, it will not accept any input, and ordinary shortcut keys (<kbd>Shift</kbd> + <kbd>F5</kbd>, <kbd>Alt</kbd> + <kbd>Tab</kbd>, etc.) will function as usual. Press Pause again to resume debugging.
 
-System keyboard
+<!-- System keyboard
 :   The system keyboard layout currently being used. You can test your
     keyboard with a different underlying layout by selecting
-    **Select System Keyboard** from the **Debug** menu.
+    **Select System Keyboard** from the **Debug** menu. -->
```
Comment (darcywong00) [inline developer/17.0/context/debug.md:71]: Is this System keyboard block no longer needed?

Comment (mcdurdin) [reply developer/17.0/context/debug.md:71]: As 'select system keyboard' was dead functionality I hid it in #7530 (16.0.91-alpha) ... so it should have been updated for 16.0 docs (too late now!)

File: developer/17.0/context/model-editor.md [hunk 2]
```diff
@@ -36,6 +36,18 @@ Word breaker
     `default`, `ascii`, or `custom`. Corresponds to the `wordBreaker`
     property in the [.model.ts source file](../reference/file-types/model-ts).
 
+Insert after word
+: *update is coming*
+
+Quotation marks
+: *update is coming*
```
Comment (darcywong00) [inline developer/17.0/context/model-editor.md:43]: I think these first two sections relate to the advanced-punctuation guide for lexical models
https://help.keyman.com/developer/17.0/guides/lexical-models/advanced/punctuation

I'll check on Keyman Developer, but I think these fields let the lexical-model author specify punctuation for:
Insert after word 
> Many languages insert a space after a word. Some languages, like Thai or Khmer, do not use spaces. To suppress the space, you may set insertAfterWord to the empty string:

quotes for keep suggestion - 
> These are the quotation marks that surround the "keep" suggestion when it's displayed in the suggestion banner.

File: developer/17.0/context/model-editor.md [hunk 3]
```diff
@@ -36,6 +36,18 @@ Word breaker
...
     property in the [.model.ts source file](../reference/file-types/model-ts).
 
+Insert after word
+: *update is coming*
+
+Quotation marks
+: *update is coming*
+
+Right-to-left script
+: *update is coming*
```
Comment (darcywong00) [inline developer/17.0/context/model-editor.md:46]: > Check this box to indicate that the lexical-model is for a right-to-left script

File: developer/17.0/context/model-editor.md [hunk 4]
```diff
@@ -36,6 +36,18 @@ Word breaker
...
+
+Language uses casing
+: *update is coming*
```
Comment (darcywong00) [inline developer/17.0/context/model-editor.md:49]: This is from keymanapp/keyman#7008 which came out of a [community forum discussion](https://community.software.sil.org/t/text-suggestion-not-working-in-some-cases/6490/6)

Check this box to indicate the lexical-model to capitalize suggestions as needed.

@jahorton prob has better guidance what to put here

File: developer/17.0/context/new-project.md [hunk 5]
```diff
@@ -2,28 +2,23 @@
...
 Creates a new keyboard or lexical model project, optionally from a template or by importing from another source.
 
-## Basic
+## Keyman keyboard
 
 Creates a keyboard project that matches the folder structure used in the Keyman [keyboards repository](https://github.com/keymanapp/keyboards) and includes all [basic files](/developer/keyboards/) needed for a keyboard project. Clicking OK will open the [New Project Parameters dialog](new-project-parameters).
 
-## Blank
+## LDML Keyboard
+*update soon*
```
Comment (darcywong00) [inline developer/17.0/context/new-project.md:14]:
```suggestion
Creates a keyboard project that matches the folder structure used in the Keyman [keyboards repository](https://github.com/keymanapp/keyboards) and includes all [basic files](/developer/keyboards/) needed for a LDML keyboard project. Clicking OK will open the [New LDML Keyboard Project Parameters dialog](new-ldml-project-parameters).
```

File: developer/17.0/context/options.md [hunk 6]
```diff
@@ -111,4 +114,14 @@ Update database
...
-    [http://www.unicode.org/ucd/](http://www.unicode.org/ucd/). This lets you update the character map with a newer version of Unicode. Keyman Developer 10.0 was released with Unicode version 10.0 data.
\ No newline at end of file
+    [http://www.unicode.org/ucd/](http://www.unicode.org/ucd/). This lets you update the character map with a newer version of Unicode. Keyman Developer 10.0 was released with Unicode version 10.0 data.
+
+## Server tab
+
+![Options dialog - Server tab](/cdn/dev/img/developer/170/ui/frmOptions_Server.png)
+
+Keyman Developer Server
+: *update soon*
```
Comment (darcywong00) [inline developer/17.0/context/options.md:124]: Maybe PR keymanapp/keyman#6073 documents features of the Keyman Developer Server which is used for testing and debugging keyboards.

Comment (darcywong00) [reply developer/17.0/context/options.md:124]: It's a lot of info, so that could go on separate PR if you want

Comment (Meng-Heng) [reply developer/17.0/context/options.md:124]: Yes, I will update this section on another PR.

File: developer/17.0/context/model-editor.md [hunk 7]
```diff
@@ -36,6 +36,26 @@ Word breaker
...
+    ```keyman
+    punctuation: {
+    insertAfterWord: "",
+    },
+    ```
+
+    [Read more](https://help.keyman.com/developer/17.0/guides/lexical-models/advanced/punctuation#toc-customizing-insertafterword)
+
+Quotation marks
+: These are the quotation marks that surround the "keep" suggestion when it's displayed in the suggestion banner. [Read about Quotation marks here](https://help.keyman.com/developer/17.0/guides/lexical-models/advanced/punctuation#toc-customizing-quotesforkeepsuggestion).
```
Comment (darcywong00) [inline developer/17.0/context/model-editor.md:51]: Maybe use relative path
```suggestion
: These are the quotation marks that surround the "keep" suggestion when it's displayed in the suggestion banner. [Read about Quotation marks here](../guides/lexical-models/advanced/punctuation#toc-customizing-quotesforkeepsuggestion).
```

File: developer/17.0/context/model-editor.md [hunk 8]
```diff
@@ -36,6 +36,26 @@ Word breaker
...
+Insert after word
+: Many languages insert a space after a word. Some languages, like Thai or Khmer, do not use spaces. To suppress the space, you may set insertAfterWord to the empty string:
+
+    ```keyman
+    punctuation: {
+    insertAfterWord: "",
+    },
+    ```
+
+    [Read more](https://help.keyman.com/developer/17.0/guides/lexical-models/advanced/punctuation#toc-customizing-insertafterword)
```
Comment (darcywong00) [inline developer/17.0/context/model-editor.md:48]: Can remove the space indentation so this isn't formatted in a code block. Also using a relative path

```suggestion
[Read more](../guides/lexical-models/advanced/punctuation#toc-customizing-insertafterword)
```

Comment (Meng-Heng) [reply developer/17.0/context/model-editor.md:48]: It didn't show that way when I ran it but I applied your suggestions 😃

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**